src/query-builder/QueryExpressionMap.ts

```diff
@@ -434,6 +434,7 @@ export class QueryExpressionMap {
         map.callObservers = this.callObservers;
         map.useTransaction = this.useTransaction;
         map.nativeParameters = Object.assign({}, this.nativeParameters);
+        map.eagerRelations = this.eagerRelations;
```

Comment:
Not sure why this line is marked as new - it is already present in next branch(introduced by #5347)

Reply:
Because this branch needs to be rebased from the latest master, most likely.